From: Simon McVittie <smcv@debian.org>
Date: Sat, 13 Feb 2021 18:25:18 +0000
Subject: reftest-compare: Fix spelling of "diff"

This is the only non-whitespace difference between the copies in
testsuite/reftests/ and testsuite/gsk/.

Signed-off-by: Simon McVittie <smcv@debian.org>
Forwarded: https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/3195
---
 testsuite/reftests/reftest-compare.c | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/testsuite/reftests/reftest-compare.c b/testsuite/reftests/reftest-compare.c
index 651eac9..34285ac 100644
--- a/testsuite/reftests/reftest-compare.c
+++ b/testsuite/reftests/reftest-compare.c
@@ -121,15 +121,15 @@ buffer_diff_core (const guchar *buf_a,
             {
               int value_a = (row_a[x] >> (channel*8)) & 0xff;
               int value_b = (row_b[x] >> (channel*8)) & 0xff;
-              guint channel_difff;
-
-              channel_difff = ABS (value_a - value_b);
-              channel_difff *= 4;  /* emphasize */
-              if (channel_difff)
-                channel_difff += 128; /* make sure it's visible */
-              if (channel_difff > 255)
-                channel_difff = 255;
-              diff_pixel |= channel_difff << (channel * 8);
+              guint channel_diff;
+
+              channel_diff = ABS (value_a - value_b);
+              channel_diff *= 4;  /* emphasize */
+              if (channel_diff)
+                channel_diff += 128; /* make sure it's visible */
+              if (channel_diff > 255)
+                channel_diff = 255;
+              diff_pixel |= channel_diff << (channel * 8);
             }
 
           if ((diff_pixel & 0x00ffffff) == 0)
